MapzenSDK/MZMapViewController.swift

```diff
@@ -582,6 +597,9 @@ open class MZMapViewController: UIViewController, LocationManagerDelegate {
     locale = l
     guard let sceneFile = styles.keyForValue(value: style) else { return }
     currentStyle = style
+    if style == .walkabout {
+      showWalkingPathOverlay = true
```

Comment:
We might not need to go through an `applySceneUpdate` and the whole process of updating the scene through Tangram here, we could set `walkingOverlayIsShowing = true` instead so `allSceneUpdates` picks it up and append the update.